kratos/python/add_constitutive_law_to_python.cpp

```diff
@@ -209,7 +228,8 @@ void  AddConstitutiveLawToPython(pybind11::module& m)
     .def("FinalizeMaterialResponsePK2",&ConstitutiveLaw::FinalizeMaterialResponsePK2)
     .def("FinalizeMaterialResponseKirchhoff",&ConstitutiveLaw::FinalizeMaterialResponseKirchhoff)
     .def("FinalizeMaterialResponseCauchy",&ConstitutiveLaw::FinalizeMaterialResponseCauchy)
-    .def("InitializeMaterial",&ConstitutiveLaw::InitializeMaterial)
+    .def("InitializeMaterial",&ConstitutiveLawInitializeMaterialStandard, py::return_value_policy::reference_internal)
+    .def("InitializeMaterial",&ConstitutiveLawInitializeMaterialProcessInfo, py::return_value_policy::reference_internal)
```

Comment:
Please use py::overload_cast (and remove the return value stuff since it is a void fct)

Reply:
Yes thanks, done!